README.md

```diff
@@ -38,7 +38,7 @@ As a developer you may want to have a local nodejs environment. That can be help
 
 You need a nodejs installation and you need to install some global packages. You just need to run the following command:
 
-    npm install -g karma-cli webpack node-sass
+    npm install -g karma-cli webpack node-sass eslint eslint-plugin-react
 
 The dev server is currently not supported, if you'd like to auto-compile your bundle, you need to run the following command:
 
```

Comment:
hmm, I think we should add a note that these tools work with phpstorm/webstorm, too